src/asm/fstack.c

```diff
@@ -42,6 +42,8 @@ size_t maxRecursionDepth;
 static unsigned int nbIncPaths = 0;
 static char const *includePaths[MAXINCPATHS];
 
+static const char *preIncludeName;
```

Comment:
```suggestion
static const char *preIncludeName = NULL;
```

Reply:
Are you sure? It's technically valid as-is since static variables are zero-initialized; I chose to leave out the `NULL` since `static struct Context *contextStack;` in the same file also doesn't have one.

Reply:
I know they are the same thing, but I prefer explicitly stating what we expect the variable to be initialised to, and leave alone variables that are "don't care". `checkpatch` used to lint against this, which is why some variables currently violate this principle.

Reply:
It's a strange day when I agree with @ISSOtm on C style... but yeah, even if static variables are default-initialized, it's better to do so explicitly.

Reply:
I also prefer explicit initializers, but would like to make that as a project-wide change in another PR.

Reply:
Ok.